Source/Core/Cartographic.js

```diff
@@ -146,6 +146,22 @@ define([
         return result;
     };
 
+    /**
+     * Returns a Cartesian3 position from a {@link Cartographic} input.
```

Comment:
`Creates a new {@link Cartesian3} instance from a Cartographic.`

No need to link to Cartographic, we're already in that class.